static/js/newsletter_signup.js

```diff
@@ -0,0 +1,95 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+'use strict';
+
+//var basketUrl = 'https://basket.mozilla.org/news/subscribe/';
+// STAGING!
+var basketUrl = 'https://basket.allizom.org/news/subscribe/';
+
+function err(e) {
+  var msg = 'An unknown error occurred. Please try again later.';
+  if (e.message != '' && e.message != undefined) {
+    msg = e.message;
+  }
+  var error = document.querySelector('.subscribe-form .error');
+  error.textContent = msg;
+  error.style.display = 'block';
+
+  if (msg == 'Invalid email address') {
+    document.getElementById('newsletter-email').parentNode.classList.add('invalid');
```

Comment:
I'm assuming we're not really concerned about older browsers for this, but for reference here's the support table for `classList`: http://caniuse.com/#search=classList